src/lib.rs

```diff
@@ -439,6 +439,28 @@ impl Client {
 
         Ok(entries)
     }
+
+    /// Flushes all existing items on the server
+    ///
+    /// This operation invalidates all existing items immediately. Any items with an update time
+    /// older than the time of the flush_all operation will be ignored for retrieval purposes.
+    /// This operation does not free up memory taken up by the existing items.
+
+    pub async fn flush_all(&mut self) -> Result<(), Error> {
+        self.conn.write_all(b"flush_all\r\n").await?;
+        self.conn.flush().await?;
+
+        let mut response = String::new();
+        self.conn.read_line(&mut response).await?;
```

Comment:
Could we reuse the `self.buf` rather than create a new string as buffer?

```
self.conn.read_buf(&mut self.buf).await?;
```

Reply:
I think you'll need to create a new string anyways to check if "OK"?